### PR TITLE
fix:  restore missing on-line examples

### PR DESCRIPTION
**Pull request by vwheeler63**
Why needed:  these changes happened to widgets without this script
being modified.
- button          (renamed)
- buttonmatrix (renamed)
- colorwheel     (removed)
- image            (renamed)
- imagebutton  (renamed)
- meter             (removed)
- lottie              (added)

This script that generates the examples pages evidently was overlooked.

@kisvegabor @kdschlosser  Gabor and Kevin, I see that the plumbing .RST and .JSON files are there under `./examples/widgets/lottie/` but can you please confirm that the `lottie` example is, in fact, ready for publication?

**>>> Note:  I need to confirm that those `lottie` examples are actually ready for publication. <<<**

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` (`astyle v3.4.12` needs to installed by running `cd scripts; ./install_astyle.sh`) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Comment by vwheeler63** [comment]
@kdschlosser @kisvegabor 

I also note that `./examples/widgets/animimg/` is not accounted for in this script.  Should I add it to the `widgets` dictionary to make it generate into the `examples.rst` file from which that page is generated?

**Comment by vwheeler63** [comment]
@kdschlosser @kisvegabor 

Good morning, Gentlemen!

I verified that the `./examples/widgets/lottie/index.rst` file is in the correct format for its reader.

Can one of you please confirm that the `lottie` example is, in fact, ready for publication?  If not, I can remove that line from the `example_list.py` and release this PR for review/merging.

**Comment by kisvegabor** [comment]
Ah, great finding! Yes, Lottie should be ready to publish! Note that, it's already available [here](https://docs.lvgl.io/master/widgets/lottie.html#load-a-lottie-animation-from-an-array)

**Comment by vwheeler63** [comment]
@kisvegabor @liamHowatt Could you two have a look at this one?

File: docs/example_list.py [hunk 1]
```diff
@@ -38,28 +38,30 @@ def process_index_rst(path):
     "widgets": "Widgets"
 }
 
+# What about animage?
```
Comment (liamHowatt) [inline docs/example_list.py]: What about it?

**Review by liamHowatt — DISMISSED** [review]

File: docs/example_list.py (hunk 1 above)
Comment (vwheeler63) [reply docs/example_list.py]: > What about it?

cc:  @kisvegabor 

Wow!  I could SWEAR I answered this Friday, but perhaps I didn't push the final button to post it....  If that's the case, I apologize for the delay.

The answer is:  Good catch!  This was an oversight.  I put the comment in there to remind me to look into "animage" and then I did but forgot to remove the comment.  I did so with the subsequent commit.

Kind regards,
Vic

**Review by kisvegabor — APPROVED** [review]
Thank you!

**Comment by vwheeler63** [comment]
@liamHowatt Would you mind having a look at this at your soonest convenience?

Kind regards,
Vic

**Review by liamHowatt — APPROVED** [review]

File: docs/example_list.py (hunk 1 above)
Comment (liamHowatt) [reply docs/example_list.py]: Sorry, I was just slow to approve. Your commit message clearly communicated the answer.

**Comment by vwheeler63** [comment]
@liamHowatt Thank you, sir!  😄

**Pull request merged**